packages/core/src/DirectoryLoader.ts

```diff
@@ -84,6 +139,7 @@ export abstract class DirectoryLoader {
 
 		if ('nodeVersions' in tempNode) {
 			for (const versionNode of Object.values(tempNode.nodeVersions)) {
+				this.injectCustomApiCallOption(versionNode.description);
```

Comment:
Do we want to inject it to all versions or just to the latest? How do we know that a version has access to a cred that supports proxy auth?

Reply:
That's a fair point as we don't have versioned credentials. I've changed it to only inject them for the latest version.